### PR TITLE
UNO 235 - Ckeditor shared stimulus

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/UNO-235
Related to: oat-sa/tao-core-ui-fe#129
Related to: https://github.com/oat-sa/tao-core/pull/2516
Created `views/js/qtiCreator/helper/ckConfigurator.js` - configs of CKEditor for shared stimulus authoring 
There is differentiation required from item authoring.  "Shared Stimulus Import" should be removed.

How to test:

- `cd tao/views/build`
- `npx grunt connect:test:keepalive`
- go to test `taoMediaManager/views/js/test/qtiCreator/helper/ckConfigurator/test.html`

### CODE REVIEW

File: views/build/grunt/test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+module.exports = function(grunt) {
```
Comment (btamas) [inline views/build/grunt/test.js:1]: license header is missing

File: views/build/grunt/test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    var watch       = grunt.config('watch') || {};
```
Comment (btamas) [inline views/build/grunt/test.js:3]: const

File: views/js/qtiCreator/helper/ckConfigurator.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Hanna Dzmitryieva <hanna@taotesting.com>
+ */
+define(['lodash', 'ui/ckeditor/ckConfigurator', 'mathJax'], function(_, ckConfigurator, mathJax) {
+  'use strict';
+
+  var _defaults = {
```
Comment (btamas) [inline views/js/qtiCreator/helper/ckConfigurator.js:24]: const

Comment (bziondik) [reply views/js/qtiCreator/helper/ckConfigurator.js]: fixed

File: views/build/grunt/test.js (hunk 2 above)
Comment (bziondik) [reply views/build/grunt/test.js]: fixed

File: views/build/grunt/test.js (hunk 1 above)
Comment (bziondik) [reply views/build/grunt/test.js:19]: fixed

**Comment by bziondik** [comment]
> When I wanted to play with it, I got an error. Should I check out something else? I followed only the instructions (btw it has a typo).
> Screen Shot 2020-05-20 at 17 58 48

Fixed style, issue about mathJax can be fixed by installing it

**Pull request merged**